test/sanity.js

```diff
@@ -94,28 +95,28 @@ transformFile('test/fixtures/test-no-svg-or-react.js', {
 transformFile('test/fixtures/test-import.jsx', {
   presets: ['airbnb'],
   plugins: [
-    '../../src/index',
+    inlineReactSvgPlugin,
   ],
 }, (err1, importResult) => {
   if (err1) throw err1;
   console.log('test/fixtures/test-import.jsx', importResult.code);
   transformFile('test/fixtures/test-require.jsx', {
     presets: ['airbnb'],
     plugins: [
-      '../../src/index',
+      inlineReactSvgPlugin,
     ],
   }, (err2, requireResult) => {
     if (err2) throw err2;
     if (importResult.code !== requireResult.code) {
-      throw new Error('Test failed: Import and require tests don\'t match');
+      throw new Error("Test failed: Import and require tests don't match");
```

Comment:
let's make this single quotes, but use a proper curly quote instead of a straight quote in `don‘t`